experimenter/experimenter/changelog/templates/changelog/test.html

```diff
@@ -0,0 +1,11 @@
+{% extends "changelog-base.html" %}
```

Comment:
regarding the file name `test.html`, we generally give test file name to those files which have some test code in it, since this is not a test template too, it is part of the `templates/changelog` you can give the appropriate name now, and keep building that page, we don't want to change name in the future